mev_inspect/models/sandwiches.py

```diff
@@ -0,0 +1,16 @@
+from sqlalchemy import Column, Numeric, String, func, TIMESTAMP, ARRAY, Integer
+
+from .base import Base
+
+
+class SandwichModel(Base):
+    __tablename__ = "sandwiches"
+
+    id = Column(String, primary_key=True)
+    created_at = Column(TIMESTAMP, server_default=func.now())
+    block_number = Column(Numeric, nullable=False)
+    sandwicher_address = Column(String(256), nullable=False)
+    frontrun_swap_transaction_hash = Column(String(256), nullable=False)
+    frontrun_swap_trace_address = Column(ARRAY(Integer), nullable=False)
+    backrun_swap_transaction_hash = Column(String(256), nullable=False)
+    backrun_swap_trace_address = Column(ARRAY(Integer), nullable=False)
```

Comment:
Why are the victim transaction hashes / trace addresses not written to the database?

Reply:
they are - they're in the join table

Reply:
https://github.com/flashbots/mev-inspect-py/pull/163/files#diff-f46030029e3258add72e9afe20c5f355d2d30d93c380505b22ea52966a6e563cR54

Reply:
Is there a specific reason to separate the victim transaction data from the bot transaction data in the database? It seems simpler to include both under the sandwiches table, rather than splitting it into two tables, and therefore removing the join on sandwich id

Reply:
because there can be multiple victims but there's only one bot